src/site/content/en/design-system/css-utilities.njk

```diff
@@ -1,12 +1,97 @@
 ---
 title: 'CSS Utilities'
+permalink: '/design-system/css-utilities/index.html'
 ---
 
 {% extends "design-system.njk" %}
 
+{% set content %}
+
+## Flow 
+
+The flow utility provides flow and rhythm between direct sibling elements.
+Where `--flow-space` is not defined: the default value is 1em, which equals
+the font size of the affected element.
+
+[More info](https://piccalil.li/quick-tip/flow-utility/)
+
+```html
+<article class="flow">
+  <p>Nullam id dolor id nibh ultricies vehicula ut id elit. Nulla vitae elit libero, a pharetra augue.</p>
+  <p>Nulla vitae elit libero, a pharetra augue. Cras justo odio, dapibus ac facilisis in, egestas eget quam.</p>
+  <p style="--flow-space: 2em">Morbi leo risus, porta ac consectetur ac, vestibulum at eros.</p>
+</article>
+```
+
+You can also use the `flow-space` generated utility that allows you to use
+the **spacing design tokens**. For example, If you want that last paragraph
+to have a `Size 2` spacing, you would change the code to the following: 
+
+```html
+<article class="flow">
+  <p>Nullam id dolor id nibh ultricies vehicula ut id elit. Nulla vitae elit libero, a pharetra augue.</p>
+  <p>Nulla vitae elit libero, a pharetra augue. Cras justo odio, dapibus ac facilisis in, egestas eget quam.</p>
+  <p class="flow-space-size-2">Morbi leo risus, porta ac consectetur ac, vestibulum at eros.</p>
+</article>
+```
+
+## Focus ring
+
+A box-shadow powered focus ring that uses a transparent outline to provide
+focus for high contrast users. The box-shadow helps to support border-radius
+elements. 
+
+```html
+<button class="[ my-cool-button ] [ focus-ring ]">My cool button</button>
```

Comment:
I'm wondering if we want to do the square bracket thing for classes or not. We don't do anything like it on d.c.c. so I can imagine it being a little confusing for team members context switching between the two sites.

Reply:
Ah yeh I'll get rid of them.